### ISSUE TITLE
Keywords in comment blocks cause parser errors

### ISSUE THREAD

**Issue by chazy** [herd/herdtools7#47]
Consider the following example used with the aarch64.cat model:

```
AArch64 MP
{
0:X1=x; 0:X3=y;
1:X1=y; 1:X3=x;
}
 P0          | P1          ;
 MOV W0,#1   | LDR W0,[X1] ;
 STR W0,[X1] | LDR W2,[X3] ; (* I like the word observed *)
 MOV W2,#1   |             ;
 STR W2,[X3] |             ;

exists (1:X0=1 /\ 1:X2=0)
```

Trying to run this litmus test will return the following error:
```
% herd7 -model aarch64.cat keyword-comment.litmus
Warning: File "keyword-comment.litmus", line 8, character 48: Lex error eof in skip_comment (in prog) (User error)
```

Removing the word 'observed' from inside the comment block resolves the issue. This seems like a priority error in the parser.

**Comment by maranget** [comment]
Hi,

Thanks for the report. The issue can be solved easily. We'll do it soon,

Thanks again.

**Comment by maranget** [comment]
Fixed by commit 15383374e31fabf51e1e28fabf20101c6e198d94.

Notice that comments are still not recognized for C  litmus tests as `(*` is easily found in C code which litmus reproduces verbatim. (See commit if interested).

Thanks again.
